powersimdata/design/investment/investment_costs.py

```diff
@@ -50,39 +48,29 @@ def calculate_ac_inv_costs(
     scenario,
     sum_results=True,
     exclude_branches=None,
-    base_grid=None,
```

Comment:
I think we should keep this. The `base_grid` here is any grid that the user would like to compare against, which is different from the base grid we have been talking about, i.e. a fresh instance of the `Grid` class without modifications by change table.

Reply:
done